browser/ui/webui/settings/brave_wallet_handler.cc

```diff
@@ -170,53 +170,3 @@ void BraveWalletHandler::SetActiveNetwork(const base::Value::List& args) {
                                  : false;
   ResolveJavascriptCallback(args[0], base::Value(result));
 }
-
-// Remove this method and pass it the non-Deprecated version instead to
-// RegisterMessageCallback once CL 3483819 lands (in Chromium 101).
-void BraveWalletHandler::AddEthereumChainDeprecated(
-    base::Value::ConstListView args) {
-  base::Value::List args_list;
-  for (const auto& arg : args)
-    args_list.Append(arg.Clone());
-  AddEthereumChain(args_list);
-}
-
-// Remove this method and pass it the non-Deprecated version instead to
-// RegisterMessageCallback once CL 3483819 lands (in Chromium 101).
-void BraveWalletHandler::SetActiveNetworkDeprecated(
-    base::Value::ConstListView args) {
-  base::Value::List args_list;
-  for (const auto& arg : args)
-    args_list.Append(arg.Clone());
-  SetActiveNetwork(args_list);
-}
-
-// Remove this method and pass it the non-Deprecated version instead to
-// RegisterMessageCallback once CL 3483819 lands (in Chromium 101).
-void BraveWalletHandler::GetAutoLockMinutesDeprecated(
-    base::Value::ConstListView args) {
-  base::Value::List args_list;
-  for (const auto& arg : args)
-    args_list.Append(arg.Clone());
-  GetAutoLockMinutes(args_list);
-}
-
-// Remove this method and pass it the non-Deprecated version instead to
-// RegisterMessageCallback once CL 3483819 lands (in Chromium 101).
-void BraveWalletHandler::RemoveEthereumChainDeprecated(
-    base::Value::ConstListView args) {
-  base::Value::List args_list;
-  for (const auto& arg : args)
-    args_list.Append(arg.Clone());
-  RemoveEthereumChain(args_list);
-}
-
-// Remove this method and pass it the non-Deprecated version instead to
-// RegisterMessageCallback once CL 3483819 lands (in Chromium 101).
-void BraveWalletHandler::GetCustomNetworksListDeprecated(
-    base::Value::ConstListView args) {
-  base::Value::List args_list;
-  for (const auto& arg : args)
-    args_list.Append(arg.Clone());
-  GetCustomNetworksList(args_list);
-}
```

Comment:
shouldn't we also remove these functions from `brave/browser/ui/webui/settings/brave_wallet_handler.h`?

Reply:
Yes, yes we should 😅 Fixed in 7203bb7a2ef6d9f6479e2c77be31d6af6bf7f7ed